uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
```

Comment:
That should be left to the implementer. I think if you see all the params each one of them is optional. One can choose to send either baro related raw info or airspeed related or both.